highfidelity/hifi
# Spectator Camera preview instructions

**Test Plan:**
1. From HMD, open the Spectator Camera app.
2. Verify that you see some instructions that read "Turn on Spectator Camera for a preview of what your monitor shows." Verify that you see a cool static GIF appear behind that text.
3. Ensure that the "MONITOR SHOWS" switch is set to "HMD Preview"
4. Turn on the Spectator Camera
5. Verify that _the instructions stay the same_ (this behavior will change soon once we get HMD preview working in the tablet)
6. Switch the "MONITOR SHOWS" switch to "Camera View"
7. Verify that the instructions are replaced by a preview of the camera view.
8. Turn the camera off. Verify that the instructions return.
9. Turn the camera on. Verify that the instructions are replaced by a preview.
10. Switch the "MONITOR SHOWS" switch to "HMD Preview". Verify that the preview is replaced with instructions (this behavior will change soon once we get HMD preview working in the tablet)
